REFERENCE.md

```diff
@@ -170,6 +170,24 @@ Windows MSI packaging and to avoid surprising upgrades.
 
 Default value: `undef`
 
+##### `package_source`
+
+Data type: `Optional[String[1]]`
+
+Source of package for installing Windows.
+Paths with http:// or https:// will be downloaded
+Paths with puppet:// or absolute filesystem paths will also be installed.
+
+Default value: `undef`
+
+##### `package_download_path`
```

Comment:
could package_download_path just use package_source if it starts with a drive letter or a network path (`//`) ?

Reply:
When the module downloads from URL it has to write that downloaded file somewhere permanently and defaults to `C:\` defined via `package_download_path`.  I figured some sites may not want installers written to `C:\` and would want to save the MSI somewhere else which can be done by changing `package_download_path`.  The value for `package_download_path` is only used when `package_source` is URL or `puppet://` as other values would mean the file already exists on filesystem.

Reply:
Updated strings documentation to clarify when `package_download_path` is used.